src/deploy/campaign/types.ts

```diff
@@ -39,13 +36,14 @@ export interface IZNSEthCrossConfig extends IZNSBaseCrossConfig {
 
 export interface IZNSZChainCrossConfig extends IZNSBaseCrossConfig {
   srcZnsPortal : string;
+  ethAdmin ?: IZNSSigner;
 }
 
 export type TZNSCrossConfig = IZNSEthCrossConfig | IZNSZChainCrossConfig;
```

Comment:
small nitpick, but it feels redundant to start every custom type with `T`, we know they are a type by the way we reference it